pkg/kubernetes/storageclass/v1alpha1/build.go

```diff
@@ -68,3 +68,13 @@ func (b *Builder) Build() (*storagev1.StorageClass, error) {
 	}
 	return b.sc.object, nil
 }
+
+// WithVolumeBind sets the volume binding mode of storageclass with provided argument.
+func (b *Builder) WithVolumeBind(bindingMode *storagev1.VolumeBindingMode) *Builder {
```

Comment:
1. Can we rename to `WithVolumeBindingMode`?
2. Can we explain in code comments what exactly is VolumeBindingMode?

Reply:
Done